server/planning/assignments/assignments.py

```diff
@@ -336,6 +338,53 @@ def send_assignment_notification(self, updates, original=None, force=False):
         else:
             event_item = None
 
+        # Create the ICS file to be added to the email usable in google calendar.
+        ical = Calendar()
+        ical.add('BEGIN', 'VCALENDAR')
```

Comment:
There's no need to add `BEGIN:VCALENDAR` specifically. The library handles this for us